packages/hoppscotch-app/pages/profile.vue

```diff
@@ -273,6 +355,39 @@ const sendEmailVerification = () => {
     })
 }
 
+const adapter = new ShortCodeListAdapter(true)
+const adapterLoading = useReadonlyStream(adapter.loading$, false)
+const adapterError = useReadonlyStream(adapter.error$, null)
+const myShortCodes = useReadonlyStream(adapter.shortCodes$, [])
+
+onLoggedIn(() => {
+  adapter.initialize()
+})
+
+const loading = computed(
+  () => adapterLoading.value && myShortCodes.value.length === 0
+)
+
+const deleteShortCode = (codeID: string) => {
+  pipe(
+    backendDeleteShortcode(codeID),
+    TE.match(
+      (err) => {
+        toast.error(`${t("error.something_went_wrong")}`)
```

Comment:
We can detect exactly what happened here. For example, if its a network error (see err.type)